views/registration.hbs

```diff
@@ -52,13 +48,15 @@
                 <button type="submit" class="login">SIGN UP</button>
 
                 <div>
-                    <p><a class='forgot-pass' href='/login'>Already have an account? Log in here.</a></p>
+                    <p>Already have an account? <a class='forgot-pass' href='/login'>Log in!</a></p>
                 </div>
 
             </form>
         </div>
 
+        <div class="houseline"></div>
+
```

Comment:
When the screen is sufficiently small (iPhone 5 in Chrome dev panel), this will display over registration form. To prevent this from happening, move the div to the top of the code, right under the navbar. (I know it's bad practice, but otherwise we have to mess with all the z-indices on the website)